build/prometheus/prometheus.yml

```diff
@@ -4,4 +4,4 @@ scrape_configs:
     scrape_interval: 5s
 
     static_configs:
-      - targets: ['localhost:9991']
+      - targets: ['notifier:9090', 'worker:9091']
```

Comment:
This is using `notifier` and `worker` instead of `localhost` because I had to disable `network_mode: host`. If you want host networking back, these will have to return to `localhost`.

Reply:
Also seems fine, I generally only poke at Grafana anyways.